ksql-engine/src/main/java/io/confluent/ksql/services/TopicValidationUtil.java

```diff
@@ -48,7 +49,9 @@ static void validateTopicProperties(
       final int actualNumPartitions,
       final int actualNumReplicas
   ) {
-    if (actualNumPartitions != requiredNumPartition || actualNumReplicas < requiredNumReplicas) {
+    if (actualNumPartitions != requiredNumPartition
+        || (requiredNumPartition != TopicProperties.DEFAULT_REPLICAS
```

Comment:
requiredNumReplicas?

Reply:
oh my! how did my unit tests not catch that... I'll add a test and fix this. Good eye

Reply:
ah I know... I had a test that failed with the correct error, but we don't check for error message hehe